README.md

```diff
@@ -8,7 +8,7 @@
 [Usage](#usage) •
 [Tutorials and Samples](#demos-tutorials-and-samples) •
 [Third-party integration](#third-party-repository-integration) •
-[NNCF Model Zoo](./docs/ModelZoo.md)
+[Model Zoo](./docs/ModelZoo.md)
```

Comment:
Shouldn't we align https://github.com/openvinotoolkit/nncf/blob/develop/docs/PyPiPublishing.md?plain=1#L23 also?

Reply:
No, the change was made to just avoid multiple lines in the header while NNCF Model Zoo is much more clear than Model Zoo, which we intended to change before.